setup.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from setuptools import setup
+from acis_taco import __version__
+try:
+    from testr.setup_helper import cmdclass
+except ImportError:
+    cmdclass = {}
+import glob
+import sys
+import os
+
+scripts = glob.glob("scripts/*")
+
+if "--user" not in sys.argv:
+    share_path = os.path.join(sys.prefix, "share", "acis_taco")
+    data_path = os.path.join(sys.prefix, "data", "acis_taco")
+
+    data_files = [(share_path, ['share/esaview.py', 'share/make_esaview_data.py']),
+                  (data_path, ['task_schedule.cfg', 'task_schedule_occ.cfg'])]
+else:
+    data_files = None
+
+setup(name='acis_taco',
+      author='Tom Aldcroft',
+      description='ACIS Earth Solid Angle package',
+      author_email='aldcroft@head.cfa.harvard.edu',
+      url='http://cxc.cfa.harvard.edu/mta/ASPECT/tool_doc/taco/',
```

Comment:
Should we leave the docs at this URL?  What installs them now?

Reply:
Agreed it would make sense to change `taco` to `acis_taco` here.

Installation is (or can be) by hand, build docs and manually rsync to the right place.  Improving this process in general would be a fine idea, but not here of course.